Mathlib/Geometry/Manifold/VectorField.lean

```diff
@@ -336,6 +362,302 @@ protected lemma _root_.MDifferentiable.mpullback_vectorField
 
 end MDifferentiability
 
+
+section ContMDiff
+
+variable [IsManifold I n M] [IsManifold I' n M'] [CompleteSpace E]
+-- the next assumptions will usually follow from the previous ones, but they are needed for
+-- the statements to make sense, so we add them here.
+[IsManifold I 1 M] [IsManifold I' 1 M']
```

Comment:
How about:
```suggestion
  -- If `1 < n` then `IsManifold.of_le` shows the following assumptions are redundant.
  -- We include them since they are necessary to make the statement.
  [IsManifold I 1 M] [IsManifold I' 1 M']
```